app/shiny.py

```diff
@@ -0,0 +1,92 @@
+from inspect import signature
+from typing import List, Optional
+
+from pydantic import BaseModel
+from shiny.session import Inputs, Outputs, Session
+from shiny.ui import (
+    _navs,
+    card,
+    card_footer,
+    card_header,
+    markdown,
+    nav_panel,
+    page_navbar,
+    tags,
+)
+
+from analyzer_interface.context import ServerCallback
+
+MANGO_ORANGE2 = "#f3921e"
+LOGO_URL = "https://raw.githubusercontent.com/CIB-Mango-Tree/CIB-Mango-Tree-Website/main/assets/images/mango-text.PNG"
+ABOUT_TEXT = markdown(
+    f"""
+
+<img src="{LOGO_URL}" alt="logo" style="width:200px;"/>
```

Comment:
Oh, one thing I'm realizing is that currently app won't work perfectly locally since we have links (this one, and the one for the roboto font in `app.py`)?

Also not urgent, just logging and can be worked out: https://shiny.posit.co/py/docs/ui-customize.html#serve-local-files